FAQ.md

```diff
@@ -98,8 +102,89 @@ This also means that if a class has a private field named `x`, code outside the
 
 ## Why is encapsulation a goal of this proposal?
 
-1. There are already good ways to provide "hidden" but not encapsulated fields, like Symbols as property names.
-
 1. Library authors have found that their users will start to depend on any exposed part of their interface, even undocumented parts. They do not generally consider themselves free to break their user's pages and applications just because those users were depending upon some part of the library's interface which the library author did not intend them to depend upon. As a consequence, they would like to have hard private state to be able to hide implementation details in a more complete way.
 
+1. While it's already possible to *model* true encapsulation using either per-instance closures or WeakMaps (see below), both approaches are unergonomic to use with classes and have memory usage semantics which may be surprising. Furthermore, per-instance closures don't allow instances of the same class to access each other's private fields (see [above](#why-does-this-proposal-allow-accessing-private-fields-of-other-instances-of-the-same-class-dont-other-languages-normally-forbid-that)), and with WeakMaps there's a non-obvious risk of exposing private data. WeakMaps are also likely to be slower than private fields could be.
+
+1. "Hidden" but not encapsulated properties are likewise already possible through the use of Symbols as property names (see below).
+
+Consensus in the committee is that we'd like a more ergonomic way to provide true encapsulation. As such, this proposal is currently moving forward with hard-private fields, letting [decorators](https://github.com/tc39/proposal-private-fields/blob/master/DECORATORS.md) or other mechanisms provide classes an opt-in escape hatch. There's still some debate about whether an escape hatch should exist by default, with some means to opt-out.
```

Comment:
Eh, you know, this was my impression at the time, but there's been more discussion on Twitter since then. I think it's better not to inflame the debate and to instead just characterize the current conclusions weakly.